babel.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  env: {
+    test: {
+      presets: [
+        [
+          '@babel/preset-env',
+          {
+            targets: {
+              node: 'current',
+            },
+          },
+        ],
+      ],
+      plugins: [
+        [
+          'module-resolver',
+          {
+            root: ['./'],
+            alias: {
+              '@': './src',
```

Comment:
This isn't the way we encourage people to reference local components. See the documentation about `src/node_modules`